frontend/src/components/Dropdown.tsx

```diff
@@ -46,11 +46,16 @@ export const Dropdown: React.FC<Props> = ({ options, selectedOption, onChange })
             ))}
           </Listbox.Options>
         </Transition>
-      </div>
+      </OptionsWrapper>
     </Listbox>
   )
 }
 
 const Img = styled.img`
   ${tw`w-5 h-5 text-gray`}
 `
+
+const OptionsWrapper = styled.div`
+  width: 336px;
```

Comment:
Let's be more consistent and define a new width breakpoint and use that `w-336px`

Reply:
fixed